CONTRIBUTING.md

```diff
@@ -15,9 +15,13 @@ Just like bug reports, take a peak at the issue tracker for duplicates before op
 To get started working on RoactRodux, you'll need:
 * Git
 * Lua 5.1
+* [Rust](https://www.rust-lang.org/tools/install) 
```

Comment:
Rust won't be necessary unless folks want to install foreman via rust. Would it be better to leave this line out and link to `foreman`'s install page instead of prescribing how to install foreman?